src/Client.php

```diff
@@ -354,4 +365,47 @@ public function watch(array $pipeline = [], array $options = [])
 
         return $operation->execute($server);
     }
+
+    private static function getVersion() : string
+    {
+        if (self::$version === null) {
+            try {
+                self::$version = PrettyVersions::getVersion('mongodb/mongodb')->getPrettyVersion();
```

Comment:
While the package-versions package caches package information when they are installed, I thought it would be best to still cache this data to ensure we're not unnecessarily causing delays on connecting.